Dockerfile

```
@@ -343,10 +348,14 @@ USER root
 #
 ################################################################################
 
-ENV HUGO_VERSION 0.85.0
+ENV HUGO_VERSION 0.101.0
 
-RUN binrc install gohugoio/hugo ${HUGO_VERSION} -c /opt/buildhome/.binrc | xargs -n 1 -I{} ln -s {} /usr/local/bin/hugo_${HUGO_VERSION} && \
-    ln -s /usr/local/bin/hugo_${HUGO_VERSION} /usr/local/bin/hugo
+RUN case "$TARGETARCH" in \
+      "arm64") HUGO_FILE="hugo_${HUGO_VERSION}_Linux-ARM64.deb" ;; \
+      "amd64") HUGO_FILE="hugo_${HUGO_VERSION}_Linux-64bit.deb" ;; \
+    esac && \
+    wget -nv --quiet https://github.com/gohugoio/hugo/releases/download/v0.101.0/${HUGO_FILE} && \
```

Shouldn't the `HUGO_VERSION` be used instead of hardcoded `v.0.101.0` in the path?

```suggestion
    wget -nv --quiet https://github.com/gohugoio/hugo/releases/download/${HUGO_VERSION}/${HUGO_FILE} && \
```